test/Test/Database/LSMTree/Normal/StateMachine.hs

```diff
@@ -1030,7 +1031,7 @@ shrinkActionWithVars ::
   -> LockstepAction (ModelState h) a
   -> [Any (LockstepAction (ModelState h))]
 shrinkActionWithVars _ _ = \case
-    New @k @v @blob conf -> [ Some $ New @k @v @blob conf' | conf' <- QC.shrink conf ]
+    New p conf -> [ Some $ New p conf' | conf' <- QC.shrink conf ]
```

Comment:
Annoying, but we can't bind type variables like this in `ghc-8.10.7`, so I added a `Proxy` argument instead. Seemed the simplest solution